pkg/session/session.go

```diff
@@ -182,6 +182,7 @@ type session struct {
 	currentCtx  context.Context // only use for runtime.trace, Please NEVER use it.
 	currentPlan base.Plan
 
+	dom   any
```

Comment:
Maybe leave a TODO to replace the usage of `any` in the future, it will reduce code maintainability.

Reply:
Ok, I add a comment here.